test/app.test.ts

```diff
@@ -204,4 +206,32 @@ describe('Plugin Tests', () => {
       app.mailer.sendPublishNotificationEmail(buildMember(lang), DEFAULT_LINK, itemName, lang);
     });
   });
+
+  describe('sendChatMentionNotificationEmail', () => {
+    it('Send chat mention notification mail with default lang', async () => {
+      setupValidateSendChatMentionNotificationMail(englishTranslations, [itemName]);
+
+      const app = await build({ plugin });
+      app.mailer.sendChatMentionNotificationEmail(
+        buildMember(),
+        DEFAULT_LINK,
+        itemName,
+        creatorName,
+      );
```

Comment:
This test does not assert a post condition, so the code path is covered but nothing is proved. Consider using an `expect` statement to check the values, e.g. inside the email, or a spy to check the calls to underlying implementations.

Reply:
@dialexo I think that the function `setupValidateSendMail` used on line 35 does the assertions. Correct me if I am wrong, but it declares a spy that then checks the values provided are in the email. I guess the implementation is a bit hidden away that way, but i think it does the tests.

Reply:
Ah i see. @pyphilia I find it pretty weird that we need to spy on promisify itself to chek the values sent to nodemailer. Can't we spy on `fastify.nodemailer.sendMail` directly, or even better get the value back here:
https://github.com/graasp/graasp-mailer/blob/ed640b93dce84f78fa6c87cfca3a42c03babdb8c/src/index.ts#L89-L90
so that we can directly spy on the decorated methods on `mailer`:
https://github.com/graasp/graasp-mailer/blob/ed640b93dce84f78fa6c87cfca3a42c03babdb8c/src/index.ts#L207-L214
Anyway I'll approve for now, but we might want to rework the tests at some point

Reply:
I see probably that would be a good idea^^